docs/source/deploy/single.rst

```diff
@@ -56,6 +56,8 @@ An example of shell command, running Iroha daemon is
 
     irohad --config example/config.sample --genesis_block example/genesis.block --keypair_name example/node0
 
+.. Note:: if you are running Iroha built with `HL Ursa support <../integrations/index.html#hyperledger-ursa>`_ please get the example keys in example/ursa-keys/
```

Comment:
Nothing is said about genesis block that should be also used instead of any other that could be used before.

Reply:
better to emphasize "example/ursa-keys/" with a style for inlined code. I guess this could be done applying backticks

Reply:
code cannot be added within the Note but it is now highlighted in _italics_ , so that's something